src/views/Login.jsx

```diff
@@ -55,27 +56,33 @@ class Login extends Component {
     } = this.state;
 
     return (
-      <div>
+      <div id="login-form">
         <h2>Login</h2>
+        <br/>
         <label htmlFor="email">Email</label>
```

Comment:
I meant the label element. The `TextField` component has floating label props: http://www.material-ui.com/#/components/text-field